far/string_utils.cpp

```diff
@@ -348,11 +348,12 @@ std::optional<std::pair<size_t, size_t>> detail::fuzzy_searcher_impl::find_in(co
 
 #include "testing.hpp"
 
-TEST_CASE("string.spaces")
+TEST_CASE("string.blanks")
 {
-	for (const auto& i: GetSpaces())
+	for (const auto& i: GetBlanks())
 	{
 		REQUIRE(std::iswblank(i));
+		REQUIRE(std::isblank(i));
```

Comment:
Redundant (and UB if `i` is not within unsigned char range anyway).
It was used in another test because of a typo.

Reply:
My bad. :(